README.md

```diff
@@ -1,5 +1,8 @@
 # 🦁 Tigris MCP Server
 
+> [!IMPORTANT]
+> Tigris now has a Remote MCP Server. [Read More about it here!](https://mcp.storage.dev)
```

Comment:
This feels slightly wrong but I'm not sure about the right messaging path for this. Consider these options:

This option emphasizes the fact that you can poke it over the internet:

```suggestion
> Tigris has a hosted MCP server with OAuth support. Read more about it on [mcp.storage.dev](https://mcp.storage.dev).
```

This option emphasizes the fact that the hosted MCP server supports multiple organizations:

```suggestion
> This MCP server works best if you are only operating in one Tigris organization. If you need to hop between organizations frequently, consider using the hosted MCP server at [mcp.storage.dev](https://mcp.storage.dev).
```

I'm torn on whether this should focus on the _implementation_ or the _effects_ of the remote MCP server vs the local one.